zerver/lib/push_notifications.py

```diff
@@ -147,12 +143,18 @@ def get_apns_context() -> Optional[APNsContext]:
     # hammered with a ton of these all at once after startup.
     loop = asyncio.new_event_loop()
 
+    async def err_func(
+        request: aioapns.NotificationRequest, result: aioapns.common.NotificationResult
+    ) -> None:
+        pass
```

Comment:
It's probably worth a comment here explaining the context around why we have an empty error function for postering.